src/prefect/cli/auth.py

```diff
@@ -228,46 +242,118 @@ def create_token(name, scope):
 
 
 @auth.command(hidden=True)
-def list_tokens():
```

Comment:
We'll have to maintain the old CLI command with a deprecation warning I think. At the very least, we'll need to keep the old command and throw an error saying what to use instead.

Reply:
Ah git diff is bad -- I see this still exists. I'll have to review it outside of GitHub.